app/models/wpcc/your_details_form.rb

```diff
@@ -6,11 +6,16 @@ class YourDetailsForm
     attr_accessor :age, :gender, :salary
     attr_accessor :salary_frequency, :contribution_preference
 
+    AGE = { minimum: 16, maximum: 75 }.freeze
```

Comment:
@norm that's a good point. I will refer to Ed and Umair for extra clarity on this point. Thanks 👍